### PR TITLE
Update OpenStreetMap.org.xml

### PR DESCRIPTION
**Pull request by koops76**
#11018

### CODE REVIEW

File: src/chrome/content/rules/OpenStreetMap.org.xml [hunk 1]
```diff
@@ -1,46 +1,29 @@
...
- see OSM_Foundation.xml for related rules -->
-<ruleset name="OpenStreetMap.org" default_off='failed ruleset test'>
-	<target host=          "openstreetmap.org" />
-	<target host=      "www.openstreetmap.org" />
-	<target host=      "api.openstreetmap.org" />
-	<target host=     "blog.openstreetmap.org" />
-	<target host=     "help.openstreetmap.org" />
-	<target host=    "lists.openstreetmap.org" />
-	<target host=     "maps.openstreetmap.org" />
-	<!-- no https:    munin.openstreetmap.org -->
```
Comment (J0WI) [inline src/chrome/content/rules/OpenStreetMap.org.xml]: This host WFM, please readd it as target.

Comment (koops76) [inline src/chrome/content/rules/OpenStreetMap.org.xml]: @J0WI `maps` or `munin`?

Comment (J0WI) [inline src/chrome/content/rules/OpenStreetMap.org.xml]: I think both

Comment (koops76) [inline src/chrome/content/rules/OpenStreetMap.org.xml]: @J0WI Done.

**Pull request merged**